packages/jest-runtime/src/index.ts

```diff
@@ -81,7 +85,13 @@ const defaultTransformOptions: InternalModuleOptions = {
 type InitialModule = Partial<Module> &
   Pick<Module, 'children' | 'exports' | 'filename' | 'id' | 'loaded'>;
 type ModuleRegistry = Map<string, InitialModule | Module>;
-type ResolveOptions = Parameters<typeof require.resolve>[1];
+
+const OUTSIDE_JEST_VM_RESOLVE_OPTION = Symbol.for(
+  'OUTSIDE_JEST_VM_RESOLVE_OPTION',
```

Comment:
```suggestion
  'OUTSIDE_JEST_VM_RESOLVE_OPTION_DO_NOT_USE_OR_YOU_WILL_BE_FIRED',
```
? 😀

Reply:
Haha :D we're allowed to use it in Jest, and the plan is that outside of Jest you can't use it even if you know the magic string, so meh :D